### PR TITLE
New package: xopen-v1.0

### PR DESCRIPTION
**Pull request by madstk1**

### CODE REVIEW

File: srcpkgs/xopen/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'xopen'
+pkgname=xopen
+version=1.0
+revision=1
+archs="i686 x86_64"
```
Comment (Chocimier) [inline srcpkgs/xopen/template]: why?

**Comment by abenson** [comment]
~~~
=> xopen-1.0_1: running do_install ...
cp -f server /usr/local/bin/xopen
cp -f client /usr/local/bin/xopen-client
~~~

This needs to be fixed.

File: srcpkgs/xopen/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+build_style=gnu-makefile
+short_desc="Simple utility to run Xorg dependant commands, from a TTY"
+maintainer="Natamo <madstk1@pm.me>"
+license="GPL-3.0-only"
+homepage="https://github.com/madstk1/xopen"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum="1ecb3e1e66a29929aae81f573e89052562a1f2bf491b5cd5c93d64e55912cae0"
+
+post_install() {
+	vlicense LICENSE
```
Comment (abenson) [inline srcpkgs/xopen/template]: This isn't needed.

File: srcpkgs/xopen/template [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'xopen'
+pkgname=xopen
+version=1.0
+revision=1
+archs="i686 x86_64"
+wrksrc="${pkgname}-${version}"
```
Comment (abenson) [inline srcpkgs/xopen/template]: This is the default, so it can be dropped

**Comment by Duncaen** [comment]
I don't think this is something that should be accepted into the repository, the implementation is unsafe and could be seen as backdoor.
You can't just run a tcp server and spawn everything it receives.

**Pull request closed without merging**